app/ui/omni-box.js

```diff
@@ -107,6 +109,34 @@ class OmniBox extends HTMLElement {
     this.forwardButton.addEventListener('click', () => {
       this.dispatchEvent(new CustomEvent('forward'))
     })
+
+    // middle mouse click paste&go
+    const win = new BrowserWindow()
+    this.input.addEventListener('paste-and-go', () => {
+      win.webContents.executeJavaScript('window.location.href = ""')
+      const rawURL = clipboard.readText()
```

Comment:
Since the `paste` event contains clipboard data, you don't need to import electron's clipboard API